src/website/views/components/forms/radio-button.pug

```diff
@@ -63,55 +63,55 @@ p.-text
   | Radio button lists are used to wrap a series of radio buttons in a list.
 
 h3 Examples
-p.-text To render a series of radio buttons in a list, apply the class <code>-list</code> to <code>m-formGroup</code>.
+p.-text To render a series of radio buttons in a list, apply the class <code>-list</code> to <code>m-form__item</code>.
 .example.-mb--3
   .-p--3
     h5 Select an option
-    ul.m-formGroup.-list(style="max-width:20rem")
-      li.m-formGroup
+    ul.m-form__item.-list(style="max-width:20rem")
```

Comment:
@jllr @janibolkvadze we may need to rethink m-form__item on the ul. it functions less like an item and more of an items/item group/group. the class itself isn't providing much. all the unique style is coming from -list.

Reply:
@jllr @janibolkvadze Marking this as resolved. I spoke with Jose and we're going to create a separate ticket for this.